lib/web_ui/dev/create_simulator.dart

```diff
@@ -0,0 +1,42 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// @dart = 2.6
+import 'dart:async';
+
+import 'package:args/command_runner.dart';
+import 'package:simulators/simulator_manager.dart';
+
+import 'safari_installation.dart';
+import 'utils.dart';
+
+class CreateCommand extends Command<bool> with ArgUtils {
+  CreateCommand() {
+    IosSafariArgParser.instance.populateOptions(argParser);
+  }
+
+  @override
+  String get name => 'create';
```

Comment:
`felt` is a general purpose dev cycle tool for the engine, so "create" would not imply anything about simulators. Consider using a more specific name, such as `felt create_simulator`, or namespacing it under a sub-command, e.g. `felt create simulator`.

Reply:
changed to `felt create_simulator`